include/arch/arm/aarch32/cortex_m/mpu/arm_mpu.h

```diff
@@ -1,61 +1,14 @@
 /*
- * Copyright (c) 2017 Linaro Limited.
```

Comment:
Don't we still need a copyright here, anyways?

Reply:
Yes, that was accidentally lost with the way the files were moved.  I've added the copyright notices back to the four compatibility headers in that directory.